index.js

```diff
@@ -6,6 +6,19 @@ function noop () {
   return undefined
 }
 
+function isArrayOfStrings (arr) {
```

Comment:
I would ignore this check - pino supports numbers and object too:

https://getpino.io/#/docs/api?id=mergingobject-object

Reply:
cool!